src/js/actions/authActions.js

```diff
@@ -0,0 +1,85 @@
+import fetch from 'fetch-everywhere';
+import {push} from 'react-router-redux';
+import urlFor from '../helpers/urlFor';
+
+const requestLogout = () => ({
```

Comment:
Methods should not be declared as constants. A constant indicates an application parameter that does not change. A method return changes based on input data. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const